modules/osd-aws-privatelink-create-cluster.adoc

```diff
@@ -0,0 +1,53 @@
+[id="osd-aws-privatelink-create-cluster.adoc_{context}"]
+= Creating your AWS PrivateLink cluster
```

Comment:
s/your /an

Reply:
@jonquilwilliams - Should I infer that we should continue to leave Rosa out of this title then? The question from @dustman9000 above?

Reply:
Thanks! Yes, I realized that my comment was in the wrong place and that you had already answered it ;-)